File: src/test/java/gameoflife/helper/SeedHelperTest.java
package gameoflife.helper;

import org.junit.Test;

public class SeedHelperTest {
    private final SeedHelper seedHelper = new SeedHelper();

    @Test(expected = RuntimeException.class)
    public void testSeedToMapWithNull() {
        seedHelper.seedToMap((String)null);
    }

    @Test(expected = RuntimeException.class)
    public void testSetBoundaryWithNull() {
        seedHelper.seedToMap(new String[0]);
    }

}

File: src/test/kotlin/gameoflife/helper/SeedHelperTest.kt
package gameoflife.helper

import org.junit.Assert.assertEquals
import org.junit.Test

class SeedHelperTest {
    @Test
    fun testSeedToMap() {
        assertEquals("{1|1=1|1, 1|2=1|2, 1|3=1|3}", SeedHelper.seedToMap("1|1, 1|2, 1|3").toString())
    }

    @Test(expected = RuntimeException::class)
    fun testSeedToMapWithNull() {
        SeedHelper.seedToMap((null as String?)!!)
    }

    @Test(expected = RuntimeException::class)
    fun testSetBoundaryWithNull() {
        SeedHelper.seedToMap(arrayOf())
    }

}
